### PR TITLE
Feature/fsar 12 landing links

### PR DESCRIPTION
**Pull request by Roxana-deeson**

### CODE REVIEW

File: src/pattern-library/components/LandingLinks/landingLinks.scss [hunk 1]
```diff
@@ -37,18 +37,24 @@
   }
 
   &__list {
-    @include grid;
-    display: flex;
-    flex-flow: row wrap;
-    justify-content: center;
-    margin-top: 2rem;
+    display: grid;
```
Comment (PeterBerryman) [inline src/pattern-library/components/LandingLinks/landingLinks.scss:40]: @Roxana-deeson this won't work in IE11, we should be using the grid-cols mixin here, so 
`@include grid-cols(2, 4)` for 2 of 4 columns on mobile
`@incliude grid-cols(4)` on tablet etc

File: src/pattern-library/components/LandingLinks/landingLinks.scss [hunk 2]
```diff
@@ -36,30 +39,46 @@
...
+      flex-direction: row;
+    }
+
 
   }
 
   &__item {
     display: flex;
     flex-direction: column;
     align-items: center;
```
Comment (c-ancia) [inline src/pattern-library/components/LandingLinks/landingLinks.scss:57]: do you still need these 3 lines?

File: src/pattern-library/components/LandingLinks/landingLinks.scss [hunk 3]
```diff
@@ -97,19 +116,19 @@
...
+  // @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+  //   &__item:not(:last-child) {
+  //     margin-right: 2rem;
+  //   }
+
+  //   &__list {
+  //     display: flex;
+  //     justify-content: center;
+  //   }
+  // }
```
Comment (c-ancia) [inline src/pattern-library/components/LandingLinks/landingLinks.scss:132]: if we don't need this anymore, you can just remove it please. :)

File: yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,11518 @@
+Arguments:
```
Comment (c-ancia) [inline yarn-error.log:1]: can you remove this file please?

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/pattern-library/components/LandingLinks/landingLinks.scss (hunk 2 above)
Comment (Roxana-deeson) [reply src/pattern-library/components/LandingLinks/landingLinks.scss:57]: yes, it's centring the svg + text in the parent container

File: src/pattern-library/components/LandingLinks/landingLinks.scss (hunk 3 above)
Comment (Roxana-deeson) [reply src/pattern-library/components/LandingLinks/landingLinks.scss]: oops

**Review by c-ancia — APPROVED** [review]

**Pull request merged**